digicampipe/tests/test_lid_ccd.py

```diff
@@ -0,0 +1,54 @@
+from digicampipe.image.sky_image import *
+from digicampipe.image.cones_image import *
```

Comment:
`from foo import *` should be avoided. It clutters global namespace and causes confusion for people who do not know by heart, what symbols the imported modules contain.

Reply:
fixed in commit lid_ccd 2dff226